README.md

```diff
@@ -3,7 +3,11 @@
 [![Total Downloads](https://poser.pugx.org/midtrans/snap/downloads)](//packagist.org/packages/midtrans/snap) 
 [![License](https://poser.pugx.org/midtrans/snap/license)](//packagist.org/packages/midtrans/snap)
 
-Midtrans ❤️ Magento! Midtrans is highly concerned with customer experience (UX). We strive to make payments simple for both the merchant and customers. With this plugin,  you can make your Magento store using Midtrans payment.
+Midtrans ❤️ Magento! Midtrans is highly concerned with customer experience (UX). We strive to make payments simple for both the merchant and customers. With this plugin, you can make your Magento store using Midtrans payment.
+
```

Comment:
I think we need to update this wording. To maybe:

> Midtrans ❤️ Magento! Midtrans strive to make payments simple for both the merchant and customers. With Midtrans Magento extension/plugin, your Magento Store can easily start accepting payments via Midtrans.

Reply:
Done updated